ui/app/src/components/ContentTypesManagement/ContentTypesManagement.tsx

```diff
@@ -58,6 +60,16 @@ export function ContentTypeManagement(props: ContentTypeManagementProps) {
         switch (e.data.type) {
           case 'CONTENT_TYPES_ON_SAVED': {
             dispatch(emitSystemEvent(contentTypeUpdated()));
+            switch (e.data.saveType) {
+              case 'saveAndClose':
+                if (embedded) {
+                  onClose?.();
+                }
```

Comment:
If the saveAndClose is available and an onClose function is sent, no reason not to call it independently of whether it is embedded or not.
```suggestion
                onClose?.();
```